### PR TITLE
[FEATURE] retrieve entity attributes using NGSI v2

### PR DESCRIPTION
**Pull request by iariasleon**
@kzangeli @crbrox 
```
                    SUMMARY:
-------------------------------------------------
  - components/ngsiv2/entities/retrieve_entity_attributes/retrieve_entity_attributes.feature >> passed: 201, failed: 0, skipped: 0 and total: 201 with duration: 17.422 seconds.
-------------------------------------------------
  Date: Wednesday, May 04, 2016 15:13:23
-------------------------------------------------
1 feature passed, 0 failed, 0 skipped
201 scenarios passed, 0 failed, 0 skipped
1247 steps passed, 0 failed, 0 skipped, 0 undefined
Took 0m17.422s
```

### CODE REVIEW

File: test/acceptance/behave/components/common_steps/entities/get_steps.py [hunk 1]
```diff
@@ -65,6 +65,18 @@ def get_an_entity_by_id(context, entity_id):
     __logger__.info("...returned an entity by id")
 
 
+@step(u'get attributes in an entity by ID "([^"]*)"')
+def get_attributes_in_an_entity_by_id(context, entity_id):
```
Comment (fgalan) [inline test/acceptance/behave/components/common_steps/entities/get_steps.py]: Remember for future PRs... it is better to split changes in PRs with .py changes and .feature changes.

Comment (iariasleon) [inline test/acceptance/behave/components/common_steps/entities/get_steps.py]: Yes,I forgot not include this file

Comment (fgalan) [inline test/acceptance/behave/components/common_steps/entities/get_steps.py]: Don't worry. Next time ;)

NTC

**Comment by jcanonav** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**